accounts/abi/argument.go

```diff
@@ -75,6 +75,15 @@ func (arguments Arguments) isTuple() bool {
 	return len(arguments) > 1
 }
 
+// Types returns the list of underlying []reflect.Type for Arguments
```

Comment:
```suggestion
// Types returns the list of underlying []reflect.Type for Arguments.
```

The first sentence of godoc is formatted differently so the full stop is important.